packages/server/migrations/201907050948001-alter-users-table.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+  up: async function (queryInterface, Sequelize) {
+    // Columns
+    await queryInterface.addColumn('Users', 'email', {
+      type: Sequelize.STRING(64),
+      allowNull: true,
+    });
+    await queryInterface.addColumn('Users', 'picture', {
+      type: Sequelize.STRING(128),
```

Comment:
why have these restrictions here? (64 / 128)? I could imagine that we may want to increase these numbers later on, especially for picture, where users may want to enter a long URL.

Reply:
We should apply restrictions on DB always. Though we could increase this limit.